F1/F1/Results/ResultsViewModel.swift

```diff
@@ -68,19 +72,19 @@ class ResultsViewModel {
                 return "No Time"
             }
         }
-
     }
 
     func fetchResults() {
-        repository?.fetchRacingResults(completion: { [weak self] result in
+        repository?.fetchRacingResults { [weak self] result in
             switch result {
             case .success(let result):
+                // TODO: Change mrData
```

Comment:
still the todo?

Reply:
nice use of trailing closure as well! :)

Reply:
Changing the name in this PR meant adding a lot more files in one PR. Going to change it in the next PR